pkg/deploy/automation_dummy_client.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package deploy
+
+import "github.com/dynatrace/dynatrace-configuration-as-code/pkg/client/automation"
+
+type dummyAutomationClient struct {
+}
+
+func (c *dummyAutomationClient) Upsert(_ automation.ResourceType, id string, _ []byte) (*automation.Response, error) {
```

Comment:
I thought an Automation Dummy already exists? @warber didn't you add one for validation?

Reply:
yes i think @jskelin just don't want to have it in `automation.go`

Reply:
Ah! Thanks, I missed the removal from `automation.go`, I thought it was in it's own file already

Reply:
yeah, although IMO there is absolutely no reason to put that in a separate file, that has in addition to it an awful name... but fine with it :man_shrugging:  :smile: